components/properties/src/props.rs

```diff
@@ -39,6 +41,41 @@ pub enum EnumeratedProperty {
     InvalidCode = -1, // TODO(#1160) - taken from ICU4C UProperty::UCHAR_INVALID_CODE
 }
 
+/// Enumerated property Bidi_Class
+///
+///
+///
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Ord, PartialOrd)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[repr(transparent)]
+#[zerovec::make_ule(BidiClassULE)]
+pub struct BidiClass(pub u16);
+impl BidiClass {
+    pub const LEFT_TO_RIGHT: BidiClass = BidiClass(0);
```

Comment:
suggestion: use `PascalCase`, not `CAPITAL_SNAKE_CASE` 

Rust uses "PascalCase" for the names of variants within an enum. So we want to do the same for the names of these constants defined within the scope of the struct.  But if you want to create a constant at the top-level, then capital snake case is preferred.

Reply:
There are Rust attributes (like C/C++ preprocessor directives) for the Clippy linter that can allow-list certain sections of code for specified types of linter issues.  Look at the other newtype structs (not enums) for enumerated properties down below for examples.

Reply:
done.